com.behringer.XAirEdit.yml

```diff
@@ -0,0 +1,40 @@
+app-id: com.behringer.XAirEdit
+runtime: org.freedesktop.Platform
+runtime-version: "19.08"
+sdk: org.freedesktop.Sdk
+command: X-AIR-Edit
+tags:
+- proprietary
+finish-args:
+- --share=ipc
+- --socket=x11
+- --share=network
+- --device=dri
+modules:
+- name: x-air-edit
+  buildsystem: simple
+  build-commands:
+  - install -d /app/bin /app/extra
+  - install apply_extra /app/bin
+  - ln -s /app/extra/X-AIR-Edit /app/bin/X-AIR-Edit
+  - install -Dm644 -t /app/share/applications com.behringer.XAirEdit.desktop
+  - install -Dm644 -t /app/share/appdata com.behringer.XAirEdit.appdata.xml
+  - install -Dm644 -t /app/share/icons/hicolor/512x512/apps com.behringer.XAirEdit.png
+  sources:
+  - type: script
+    dest-filename: apply_extra
+    commands:
+    - tar xvf X-AIR-Edit_LINUX_X64_V1.5.tar.gz
+  - type: file
+    path: com.behringer.XAirEdit.desktop
+  - type: file
+    path: com.behringer.XAirEdit.appdata.xml
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html